site/pages/api/pledge/index.ts

```diff
@@ -26,6 +26,7 @@ export default async function handler(
           );
         }
 
+        await res.unstable_revalidate(req.body.urlPath);
```

Comment:
nice, this is rad. We will have to circle back after we address #714 because it was made stable in 12.2 🎉

Reply:
Sweet, I noticed that too because `res.revalidate` wouldn't work 😆